src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardSamlBundle/DependencyInjection/Configuration.php

```diff
@@ -45,10 +45,10 @@ public function getConfigTreeBuilder()
     private function appendSessionConfiguration(NodeBuilder $childNodes)
     {
         $childNodes
-            ->scalarNode('administrator_team')
+            ->arrayNode('administrator_teams')
+                ->info('All users in these teams get the administrator role')
                 ->isRequired()
-                ->defaultValue('urn:collab:org:surf.nl')
-                ->info('All users in this team get the administrator role')
+                ->scalarPrototype()->end()
```

Comment:
Sure about removing 'a sensible default' value?

Reply:
Yes, setting the default value previously was a redundancy, as it is already set in the `parameters.yml.dist` and also guaranteed to be set when deploying using Ansible. Also, the `arrayNode` does not provide the possibility to set a default value.

Reply:
Right, agreed